cloudinit/cmd/devel/shell_script_per_boot.py

```diff
@@ -0,0 +1,25 @@
+# part-handler
+# vi: syntax=python ts=4
+
+import os
+from cloudinit import log
+from cloudinit import util
+
+LOG = log.getLogger(__name__)
+
+
+def list_types():
+    # return a list of mime-types that are handled by this module
+    LOG.debug("in shell_script-per-boot.list_types() ...")
+    return(["text/x-shellscript-per-boot"])
+
+
+def handle_part(data, ctype, script_path, payload):
+    if script_path is not None:
+        LOG.info("in shell_script-per-boot.handle_part() ...")
+        LOG.debug("script_path=%s", script_path)
```

Comment:
Let's drop the verbose logging in these handlers; util.write_file() already logs which file it writes.